mappings/net/minecraft/client/font/GlyphAtlasTexture.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/class_380 net/minecraft/client/font/GlyphAtlasTexture
+	FIELD field_21690 textLayer Lnet/minecraft/class_1921;
+	FIELD field_21691 textSeeThroughLayer Lnet/minecraft/class_1921;
```

Comment:
maybe `transparentTextLayer`?

Reply:
`seeThrough` is sourced from the layer name: https://github.com/FabricMC/yarn/blob/20w29a/mappings/net/minecraft/client/render/RenderLayer.mapping#L32

Reply:
maybe `seeThroughTextLayer`? Honestly we shouldn't fluctuate between prefix and suffix because mojang do

Reply:
@liach done.